spec/requests/odata/root_json_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "root json" do
+  include_context "odata"
+
+  let(:path) { mission_api_route }
+
+  context "with no forms" do
+    it "renders as expected" do
+      expect_output({
+        "@odata.context": "http://www.example.com/en/m/#{get_mission.compact_name}/odata/v1/$metadata?mode=m",
+        value: []
+      }.to_json)
+    end
+  end
+
+  context "with several forms" do
+    include_context "odata_with_forms"
+
+    it "renders as expected" do
+      expect_output({
+        "@odata.context": "http://www.example.com/en/m/#{get_mission.compact_name}/odata/v1/$metadata?mode=m",
+        value: [
+          {name: "Responses: Sample Form 1", kind: "EntitySet", url: "Responses: Sample Form 1"}
```

Comment:
The `url` doesn't look like a url?

Reply:
This is a weird part of the OData protocol; `url` here represents the sub-path from the current route. So you hit `/odata/` to get this data, and append to get `/odata/Responses: Sample Form 1` as the route that returns those responses.

I've tested this several ways and Power BI handles URL encoding nicely (symbols/spaces are not an issue).